webapp/src/components/sidebar_right/github_items.jsx

```diff
@@ -34,9 +35,24 @@ function GithubItems(props) {
         let number = null;
 
         if (item.number) {
+            let iconType;
+
+            // item is a pull request
+            if (item.pull_request) {
+                iconType = GitPullRequest;
+            } else {
+                iconType = IssueOpened;
+            }
             number = (
                 <strong>
-                    <i className='fa fa-code-fork'/>{' #' + item.number}
+                    <div style={{...style.icon}}>
```

Comment:
Nit: wouldn't it make more sense here a `<span>` instead of a `<div>`?

Reply:
Good suggestion :+1: